app/components/ui/atoms/icons/Search.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default props => (
+  <svg viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I see that https://github.com/smooth-code/svgr has MIT license so I guess we don't have to give credit to them, or should we?